src/Avalonia.Animation/Animation.cs

```diff
@@ -321,58 +326,69 @@ private static Type GetAnimatorType(AvaloniaProperty property)
         }
 
         /// <inheritdoc/>
-        public IDisposable Apply(Animatable control, IClock clock, IObservable<bool> match, Action onComplete)
+        public IDisposable Apply(Animatable control, IClock? clock, IObservable<bool> match, Action? onComplete)
         {
             var (animators, subscriptions) = InterpretKeyframes(control);
             if (animators.Count == 1)
             {
-                subscriptions.Add(animators[0].Apply(this, control, clock, match, onComplete));
+                var subscription = animators[0].Apply(this, control, clock, match, onComplete);
+                
+                if (subscription is not null)
```

Comment:
@jmacato: not 100% sure if this is correct, if the animator returns a `null` should be be an error?

Reply:
This subscriptions list is passed to the CompositeDisposables ctor, which throws if there is a null disposable inside 
https://github.com/dotnet/reactive/blob/main/Rx.NET/Source/src/System.Reactive/Disposables/CompositeDisposable.cs#L107

So I would assume previous logic would always throw argument null exception if Apply returns null. I guess we can safely remove nullability from Apply and throw as well. Or return empty disposable instead.

Reply:
Makes sense - @jmacato does this sound right to you?

Reply:
yes this looks correct

Reply:
@grokys i think we should remove the nullability for Apply method and just make TransformAnimator to log and just return an empty disposable